pkg/services/object/put/distributed.go

```diff
@@ -154,7 +155,7 @@ func (t *distributedTarget) Close() (oid.ID, error) {
 		return oid.ID{}, err
 	}
 
-	if t.localNodeInContainer {
+	if t.localNodeInContainer && t.metainfoConsistencyAttr != "" {
```

Comment:
could u pls remind is `t.metainfoConsistencyAttr != ""` == enabled meta-on-chain?

Reply:
in practice yes. but non-zero also varies so i added a `switch` later